### PR TITLE
Changelog for v3.8.2

### PR DESCRIPTION
**Pull request by vinay-gopalan**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,12 @@
+## 3.8.2 (August 11, 2022)
+IMPROVEMENTS:
+* Add bootstrap field to Consul backend resources: ([#1571](https://github.com/hashicorp/terraform-provider-vault/pull/1571))
+* Add `data` field to KV data sources: ([#1577](https://github.com/hashicorp/terraform-provider-vault/pull/1577))
+* Update kvv1 test to use AssertVaultState(): ([#1574](https://github.com/hashicorp/terraform-provider-vault/pull/1574))
```
Comment (benashz) [inline CHANGELOG.md:5]: Probably can drop this.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+
+BUGS:
+* fix: remove unnecessary nesting of secret data for KV-V1 secrets: ([#1570](https://github.com/hashicorp/terraform-provider-vault/pull/1570))
+
+BACKWARDS INCOMPATIBILITIES / NOTES:
```
Comment (benashz) [inline CHANGELOG.md:10]: Should probably come at the beginning and I think we can just call it `NOTES` or `IMPORTANT NOTES`

**Review by benashz — APPROVED** [review]
Some minor suggestions, otherwise :+1:

File: CHANGELOG.md (hunk 2 above)
Comment (vinay-gopalan) [reply CHANGELOG.md]: Updated, thanks!

**Pull request merged**